src/web/mjs/connectors/templates/Lezhin.mjs

```diff
@@ -35,31 +33,57 @@ export default class Lezhin extends Connector {
             }
         };
     }
-
     async _initializeAccount() {
+        if(this.userID) {
+
+            //check if user disconnected
+            const uri = new URL(this.url);
+            const checkscript = `
+            new Promise((resolve, reject) => {
+                setTimeout(() => {
+                    try {
+                        resolve(__LZ_CONFIG__);
+                    }
+                    catch(error) {
+                        reject(error);
+                    }
+                },
+                5000);
+            });
+            `;
+            const request = new Request(uri, this.requestOptions);
+            const data = await Engine.Request.fetchUI(request, checkscript);
+            if (!data.token) {
+                this.requestOptions.headers.delete('Authorization');
+                this.userID = '';
+            }
+        }
+
         if(this.userID || !this.config.username.value || !this.config.password.value) {
             return;
         }
+        const password = this.config.password.value.replace("'", "\\'"); //escape the password, because if it contains a single quote the script will fail
         let script = `
-            new Promise((resolve, reject) => {
-                try {
-                    if($('#log-nav-email').length) {
-                        return resolve();
-                    }
-                    const form = $('form#email');
-                    form.find('input#login-email').val('${this.config.username.value}');
-                    form.find('input#login-password').val('${this.config.password.value}');
-                    $.ajax({
-                        type: 'POST',
-                        url: form.prop('action'),
-                        data: form.serialize(),
-                        success: resolve,
-                        error: reject
-                    });
-                } catch(error) {
-                    reject(error);
+        new Promise((resolve, reject) => {
+            try {
```

Comment:
The `try/catch` block might be redundant as a promise will automatically `reject` when a non-caught error occurs (in synchronous code)

Reply:
I just mimicked other promisse, but that makes sense.